core/server/common/src/main/java/alluxio/cli/validation/ClusterConfConsistencyValidationTask.java

```diff
@@ -58,7 +58,13 @@ public TaskResult validate(Map<String, String> optionMap) throws InterruptedExce
       propertyNames.addAll(props.stringPropertyNames());
     }
     for (String propertyName : propertyNames) {
-      PropertyKey propertyKey = PropertyKey.fromString(propertyName);
+      PropertyKey propertyKey = null;
+      try {
+        propertyKey = PropertyKey.fromString(propertyName);
+      } catch (IllegalArgumentException e) {
```

Comment:
is there a better way to check whether the key is valid?

Reply:
replaced with `propertyKey.isValid()`